lib/osv/models.py

```diff
@@ -310,24 +311,29 @@ def _pre_put_hook(self):
 
     search_indices.update(self._tokenize(self.id()))
 
-    if self.affected_packages:
-      self.project = [
-          pkg.package.name for pkg in self.affected_packages if pkg.package.name
-      ]
-      self.ecosystem = [
-          pkg.package.ecosystem
-          for pkg in self.affected_packages
-          if pkg.package.ecosystem
-      ]
-      self.purl = [
-          pkg.package.purl for pkg in self.affected_packages if pkg.package.purl
-      ]
-
-      for project in self.project:
-        search_indices.update(self._tokenize(project))
-
-      for ecosystem in self.ecosystem:
-        search_indices.update(self._tokenize(ecosystem))
+    for affected_package in self.affected_packages:
```

Comment:
Do we want the bail out condition on self.affected_packages ? previously you had "if self.affected_packages:"

Reply:
Not needed since this is a ndb repeated property so the default value is always `[]`. The if condition was previously used to distinguish between pre-1.0 and 1.0 schema versions during migraiton.